command/agent/config/config.go

```diff
@@ -107,9 +112,35 @@ func LoadConfig(path string, logger log.Logger) (*Config, error) {
 		return nil, errwrap.Wrapf("error parsing 'cache':{{err}}", err)
 	}
 
+	err = parseCache(&result, list)
```

Comment:
```suggestion
	err = parseVault(&result, list)
```

Reply:
I am genuinely confused about this. It was working without this change, I don't know how.